packages/expo-dev-launcher/android/src/rn74/main/com/facebook/react/runtime/NonFinalBridgelessDevSupportManager.java

```diff
@@ -46,150 +46,97 @@
-        mReactHost = host;
-    }
-
-    @Override
-    protected String getUniqueTag() {
-        return "Bridgeless";
-    }
-
-    @Override
-    public void loadSplitBundleFromServer(
-            final String bundlePath, final DevSplitBundleCallback callback) {
-        fetchSplitBundleAndCreateBundleLoader(
-                bundlePath,
-                new CallbackWithBundleLoader() {
-                    @Override
-                    public void onSuccess(final JSBundleLoader bundleLoader) {
-                        mReactHost
-                                .loadBundle(bundleLoader)
-                                .onSuccess(
-                                  (Continuation<Boolean, Void>) task -> {
-                                      if (task.getResult().equals(Boolean.TRUE)) {
-                                          String bundleURL =
-                                                  getDevServerHelper().getDevServerSplitBundleURL(bundlePath);
-                                          ReactContext reactContext = mReactHost.getCurrentReactContext();
-                                          if (reactContext != null) {
-                                              reactContext.getJSModule(HMRClient.class).registerBundle(bundleURL);
-                                          }
-                                          callback.onSuccess();
-                                      }
-                                      return null;
-                                  });
-                    }
-
-                    @Override
-                    public void onError(String url, Throwable cause) {
-                        callback.onError(url, cause);
-                    }
-                });
-    }
-
-    @Override
-    public void handleReloadJS() {
-        UiThreadUtil.assertOnUiThread();
-
-        // dismiss redbox if exists
-        hideRedboxDialog();
-        mReactHost.reload("BridgelessDevSupportManager.handleReloadJS()");
-
-        // 0.74 workaround for https://github.com/facebook/react-native/commit/524e3eec3e73f56746ace8bef569f36802a7a62e
-        isPackagerRunning(isMetroRunning -> {
-          if (!isMetroRunning) {
-            String bundleURL = getDevServerHelper().getDevServerBundleURL(Assertions.assertNotNull(getJSAppBundleName()));
-            reloadJSFromServer(bundleURL, () -> {
-              UiThreadUtil.runOnUiThread(getReactInstanceDevHelper()::onJSBundleLoadedFromServer);
-            });
-          }
-        });
-    }
-
-    private static ReactInstanceDevHelper createInstanceDevHelper(final ReactHostImpl reactHost) {
-        return new ReactInstanceDevHelper() {
-            private WeakReference<ReactSurfaceImpl> logBoxSurface = new WeakReference<>(null);
-
-            @Override
-            public void onReloadWithJSDebugger(JavaJSExecutor.Factory proxyExecutorFactory) {
-                // Not implemented
+public class NonFianlBridgelessDevSupportManager extends DevSupportManagerBase {
```

Comment:
```suggestion
public class NonFinalBridgelessDevSupportManager extends DevSupportManagerBase {
```

maybe original name is correct !?

also i think we can remove the rn74 files now.